### ISSUE TITLE
multiple endpoints

### ISSUE THREAD

**Issue by joephalex** [vpnhood/VpnHood#242]
I would like to check regarding multiple endpoints. when using -ep and adding multiple IPs separated with comma

When trying to connect to the server using the client would it check which endpoint IP it be connected to or it's random?

sudo /opt/VpnHoodServer/vhserver gen -ep IP1,IP2

The reason is that in the case of one of the IPs was blocked the client will not be able to connect to the server so during the connecting it will try the next IP to check whether it could connect to.

If so could we mix IPv4 with IPv6?

**Comment by trudyhood** [comment]
Hi, Thank you for sharing your thought with us.

You can not set multiple -ep in FileAccessServer, but it is available via VpnHood Access Server. In VpnHood Access Server, you can mix IPv6 and IPv4, and the client will select one depending on its network IP version support. However, I can't see any benefit in adding multiple IPs in a token in case of blocking. If someone blocks one IP, they definitely know about the other IPs too. A blocker who has access to your token will block your server IP and rerun your access key until your last IP gets blocked. 
Multiple IP is useful for load balancing or in case of a server failure. 

In short:
- We haven't added any command line to support multiple IPs in File Access Server
- The client will choose one depending on his network IP version. VpnHood client does not try to connect via other IPs in the token if one fails.

Feel free to add any more comments on this closed issue.
